Docs/source/developers/faq.rst

```diff
@@ -70,3 +70,20 @@ What does const int ``/*i_buffer*/`` mean in argument list?
 This is often seen in a derived class, overwriting an interface method.
 It means we do not name the parameter because we do not use it when we overwrite the interface.
 But we add the name as a comment ``/* ... */`` so that we know what we ignored when looking at the definition of the overwritten method.
+
+
+What is Pinned Memory?
+----------------------
+
+We need pinned aka "page locked" host memory when we:
+
+- do async copies from/to host<->device
```

Comment:
Thanks for the PR, this is very useful to have. A comment on the English - perhaps this might be more succinct

- do asynchronous copies between the host and device

Reply:
definitely, thanks Dave!